### ISSUE TITLE
[Publications] review text size/colors to improve a11y

### ISSUE THREAD

**Issue by sabrinang** [mozilla/foundation.mozilla.org#5950]
Review smaller text areas on publications and recommend changes to improve a11y. Some examples may be:
<img width="851" alt="image" src="https://user-images.githubusercontent.com/2492510/102817911-781fe680-4385-11eb-8f7d-042c91ba2639.png">
<img width="497" alt="image" src="https://user-images.githubusercontent.com/2492510/102817960-8ec63d80-4385-11eb-9ec4-29ff907f240c.png">
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/2492510/102817990-a00f4a00-4385-11eb-84b7-54723ef1d9ae.png">

**Comment by sabrinang** [comment]
@kristinashu R? 
Let me know if any other areas on publications that I may of missed that a11y/contrast could be improved?

**Comment by kristinashu** [comment]
These look good Sabs! Curious if devs are able to test and flag any possible additional a11y issues?

**Comment by mmmavis** [comment]
For colour contrast, there should be many online checkers available. There are plugins and modules that we can check for other a11y issues but these tools will require things to be implemented first so they can scan through the page and code for possible issues. They won't work on static images.

**Comment by kristinashu** [comment]
Ok thanks Mavis! 

R+ on this Sabs!

**Comment by sabrinang** [comment]
Great thank you both! Created dev issue here https://github.com/mozilla/foundation.mozilla.org/issues/6285
